scripts/docker/cross_compile/cross_compile.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+#
+# Copyright (c) 2017, United States Government, as represented by the
+# Administrator of the National Aeronautics and Space Administration.
+# 
+# All rights reserved.
+# 
+# The Astrobee platform is licensed under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with the
+# License. You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+# This image is split into two, because we need to copy over the toolchain
+# folder and rootfs folder. Because docker can only access files in the
+# build context, and we don't want it to scan the entire computer by
+# chosing a bigger build context, this is the easiest solution. This also
+# allows the user to place the folders freely.
+
+# Base docker image copies over the toolchain and rootfs, resulting from
+# the NASA_INSTALL.md instructions. Also installs minimum dependencies
+echo "Build context for base: "${ARMHF_CHROOT_DIR/../}
+docker build $ARMHF_CHROOT_DIR/../ -f scripts/docker/cross_compile/astrobee_base_cross.Dockerfile -t astrobee/astrobee:base-cross
```

Comment:
In other scripts we use the ARMHF_TOOLCHAIN variable and these two don't have to be in the same place... could we make the docker respect this as well?

Reply:
also we are assuming that the chroot dir is called ${ARMHF_CHROOT_DIR}/../arm_cross which the other tools do not assume

Reply:
I guess this would make us have two base files with two contexts. Another option could be we would have one environment variable for both of these and a script / instructions to check them out in the right way, and change everything else to only use this variable.

Reply:
The variables are extensively used throughout the code, it makes sense to keep them separate for readability. I'll just add another build context.

Reply:
I've corrected this :) Thank you!!